lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -15,6 +15,35 @@ def self.value_of(key)
         end
       end
 
+      # Combines potentially two properties into one.
+      # Given the example: 
+      #    :disable_net_http and :prepend_net_instrumentation
+      #    if :disable_net_http is true, then returned value is "disabled"
+      #    if :prepend_net_instrumentation is false, then returned value is "chain"
+      #    otherwise, "auto" is returned.
+      #
+      # Intent is: 
+      #     - if user sets disable_xxx property, then don't instrument
+      #     - if user set prepend to `false` then we use method_alias chaining
+      #     - auto, when returned means, try to use prepend unless conflicting gems discovered
+      #
+      def self.instrumentation_value_of(disable_key, prepend_key=nil)
```

Comment:
hmmm I'm confused about this diff, because the target branch, prepend_instrumentation, has this change on it already.